File: C++/lesson _4_loop/exercise/leap_year.cpp
#include<iostream>
using namespace std;
int main(){
int year;
cout<<"enter a year"<<endl;
cin>>year;
if (year%400==0 && year%100==0 && year%4==0)
{
    cout<<" Leap Year."<<endl;

}
else if(year%400!=0 && year%100==0 && year%4==0)
{cout<<"Not a Leap Year."<<endl;}
else if(year%400==0 && year%100!=0 && year%4==0)
{cout<<"Leap Year."<<endl;}

else if(year%400!=0 && year%100!=0 && year%4==0)
{cout<<"Leap Year"<<endl;}
else
{
    cout<<"Not a Leap Year.";
}

    return 0;
}

File: Python/leap_year.py
year = int(input("Enter a year : "))

if year%4==0:
    if year%100==0:
        if year%400==0:
            print("year is a leap year as well as century year.")
        else:
            print("year is century year but not leap year.")
    else: 
        print("year is a leap year.")
else:
    print("not a leap year.")